dali/test/python/test_operator_reduce.py

```diff
@@ -47,7 +45,7 @@ def __init__(self, data_type):
         super().__init__(data_type)
         self._data = [
             [
-                np.array([ 1,  2,  3,  4], dtype=self._data_type),
+                np.array([1,  2,  3,  4], dtype=self._data_type),
```

Comment:
```suggestion
                np.array([ 1,  2,  3,  4], dtype=self._data_type),
```

Reply:
Linter complains about leading space after `[`. I decided on aligning the remaining columns to the right while leaving the first one aligned to the left.